config.py

```diff
@@ -21,10 +21,10 @@
 if not os.environ.get('COBOT_PREFIX'):
     BOT_ALT_PREFIXES = ('cobot ', )
 
-BOT_ADMINS = os.environ.get('BOT_ADMINS', '').split() or ('meetmangukiya', )
+BOT_ADMINS = os.environ.get('BOT_ADMINS', '').split() or ('*@localhost', )
```

Comment:
According to errbot specs, for text backend, it should be `@localhost` or `#localhost`

Reply:
No it isnt.  See `config-template.py`

Reply:
See also https://github.com/errbotio/errbot/issues/1135

Reply:
okk :+1:

Reply:
Sorry, you were right for the `Text` backend.  We should fix that here quickly, and upstream, as it is silly that `Text` doesnt accept the normal syntax when it shouldnt care about the bot admin anyway as it isnt on a shared service.